### DISCUSSION TITLE
(Unofficial) ToonSquid Communities

### DISCUSSION THREAD

**Discussion by keiwando** [keiwando/toonsquid#548] [General]
For anyone interested in finding more places to discuss ToonSquid or get help from the community, here are links to some unofficial ToonSquid communities that are fully run and moderated by community members:

- [Discord "ToonSquid Unofficial"](https://discord.com/invite/83C3qcmPmQ)
- [Reddit "r/ToonSquidAnimators"](https://www.reddit.com/r/ToonSquidAnimators/)
- [Twitter / X "ToonSquid Community"](https://twitter.com/ToonsquidCommu)
- [Instagram "ToonSquid Community"](https://www.instagram.com/toonsquidcommunity/?next=%2F)

To reiterate: I am not involved in any of these communities. 
If you have feature requests, bug reports or questions that I should see, either post them in this [discussion forum](https://toonsquid.com/discussions) or use the contact information at https://toonsquid.com/support/.

**Comment by otaworks** [comment]
Thank you, Keiwando! Please hang in there and take much time needed to make 2.0.
We all truly appreciate your efforts.

**Reply by Gottafindaname** [reply]
V.2 will be so amazing!

**Comment by TempleLionStudios** [comment]


**Reply by Lioncub901** [reply]
I joined that discord, before but I left. I really feel you should not pressure or try convince people to use ToonSquid and just let them use the app they want. It would just make people feel uncomfortable or left out. I feel everyone should be left alone unless they ask about what you use.

**Reply by TempleLionStudios** [reply]
No one is pressuring or convincing anyone. But we share knowledge and experience. That is important. 

I'm sorry you had a feeling of pressure. I wish you would have said this at the moment, so we could have clarified intents then. 

Many feel extremely frustrated with procreate dreams and other apps. That's why we merely suggest an app that isn't frustrating to help each other. I hope you see the difference now. 😊 

I invite you back, not that you know the true intent.

**Reply by Gottafindaname** [reply]
> I joined that discord, before but I left. I really feel you should not pressure or try convince people to use ToonSquid and just let them use the app they want. It would just make people feel uncomfortable or left out. I feel everyone should be left alone unless they ask about what you use.

I am part of this server and people there are the nicest people of any server ive been a part of. Especially TLS. So i for one do not agree with you at all and i kinda feel like you are just being a bit rude lioncub….

**Reply by TempleLionStudios** [reply]
Thanks. But it's okay 😊 I think it's more a misunderstanding. 

But I can struggle to understand why one would discuss it here instead of just sending a dm 🤷🏽‍♂️😅 

Anyways, you are welcome to rejoin LionCub901! ♥️😊

**Reply by Animartin95** [reply]
I joined this server about three months ago and I no one pressured me or anyone else. Instead is was shown how great ts is and I happily “switched sides”. 

I wish I had more time to animate right now to participate more, but i have to say, everyone have been very welcoming and friendly.

**Reply by TempleLionStudios** [reply]
> I joined this server about three months ago and I no one pressured me or anyone else. Instead is was shown how great ts is and I happily “switched sides”. 
> 
> 
> 
> I wish I had more time to animate right now to participate more, but i have to say, everyone have been very welcoming and friendly. 

Glad you feel that way Martin ♥️😊

**Reply by Lioncub901** [reply]
Maybe “pressure” is a strong word. What I am saying is that if you are going to make a place where a diverse people come and use a server I would not recommend you telling/nudging them to use a particular app because it might make them feel uncomfortable. You said, “ I also run an "iPad Animators" discord server, but honestly 95% of us use Toonsquid and I tell everyone about TS 😉 (invite link below).” This is what I’m saying. It’s OK to tell people about ToonSquid, but I don’t feel like it should be in other people’s channels ,maybe general. These images are from procreate dreams channel or another apps

https://discord.com/channels/1158441909895823391/1220686045947891792/1239229457126916187

I am not saying what you said in these messages are evil or bad, but if you are going to make a server where lot of people of diverse apps come. I would recommend you not to nudge people to use a particular app especially in their app’s channel because it defeats the purpose of the server (iPad animators). Especially that you’re the admin of the server. Lots of people don’t really speak out when they feel uncomfortable but just leave (me to). It’s like if you made a group for people who love sports with different channels for each sport, but in each channel (that is not soccer) the admin says “have you heard or soccer you might like it.” it is not bad to say that but it is not so great of a love especially for those you don’t like soccer or play the sport. Same goes for ToonSquid. I love ToonSquid, but I feel I should only show that love in appropriate places not in other apps server. 

Also @Animartin95:

I am not a “little kid” (I am 18) and am not trying to trash talk your server. Just give some small advice that might help the server run better.

No hate to you @TempleLionStudios . I just want to give some advice so other people who join don’t feel the ToonSquid community is an overzealous set of people and that every user feels welcome no matter what app they use. I am loving your animations. I am subscribed to your YouTube channel. Have a nice day.

**Reply by Gottafindaname** [reply]
> Maybe “pressure” is a strong word. What I am saying is that if you are going to make a place where a diverse people come and use a server I would not recommend you telling/nudging them to use a particular app because it might make them feel uncomfortable.

What I can’t respect is you saying it here where TLS is sharing the server! That’s what makes your post rude! Have the decency to tell it to the guy and not spread rumors in a thread like this! 

Also “nudge” is a strong word! Your point falls flat when you say it here instead. 

And if you didn’t mean “pressure” then DON’T use that word! I’m reporting you!

**Reply by TempleLionStudios** [reply]
@Lioncub901 
You feel uncomfortable sharing feedback with me, but you don't seem very uncomfortable saying it on a thread encouraging like-minded people to join? 

That is a weird choice to me. I am always open to critique and I can completely understand your view. If you would have said it I would have listened. But when you all of a sudden appear out of nowhere trying to paint a negative picture and couldn't come to me with feedback, it appears very unprofessional and unnecessary, my friend 🙏🏽 I hope you can see that part. 

Also, sharing conversations from the server is against the community rules. I suggest you remove them. 

As I said. You can reach me on discord in a dm. This is not the platform for discussing your opinions.

**Reply by Lioncub901** [reply]
@Gottafindaname 

Fair point, but there really wasn’t any other place to share neutrally. If I say it in the discord server it would seem like an attack and I really don’t like DMing strangers (someone I haven’t personally talked to). Please join the server if you want to, i’m just trying to give some advice. Also, I don’t really like “nice people,” because they end up reporting over some criticism/advice which is not cool in my opinion. You can say what you want and I will not report you because I believe in free speech.

@TempleLionStudios 

I did not say I was uncomfortable sharing my point of view. But it would not be right to say it in the server, especially if people love it and just don’t see my view. I apologize this might not be the most appropriate area to state, but I believe it would help improve the environment of the community and make everyone feel welcome no matter what app they use.

I did not know there was some rule against sharing public chats. I don’t see why it’s wrong since you’re trying to make people go to the same server so they can see it themselves. I’m not trying to expose the people, but give evidence of what I’m trying to say so you can understand. I might resend them, but with blurring out the user (Thanks).

there is nothing particularly wrong with the server at all. Just trying to make everyone feel welcome not just ToonSquid people. Wish you all the best. Please don’t be discouraged from using the server.

**Reply by TempleLionStudios** [reply]
@Lioncub901 

Well sending me a dm explaining yourself would have sufficed. 

You comment doesn't make much sense since you are sharing it here anyways. 

You got your point across and I will consider your view. 

Yes people can join and respect the rules. But as it states in the rules, sharing anything said in the server is not allowed. 
I take this break seriously. 

Your comments are no longer needer in this thread.

**Reply by Animartin95** [reply]
@Lioncub901 18? Well that explains it all. 
I agree with the others. You lack respect, and rule-breaking is just making it even worse!!

If you do not remove this, I will also have to report you!

**Reply by Lioncub901** [reply]
@TempleLionStudios 
Fair enough I remove all the images and replaced it with one message that way I am not sharing the direct messages but just a link for the evidence I was talking about. That way outsider won’t see it but only those who are in the group.

For those who want me to delete this. I won’t. There is nothing wrong with trying to give advice to help improve your server (to make everyone feel welcome not just ToonSquid users). Go ahead and report me, I don’t care. I still like ToonSquid. I hope you guys have a very nice day.

**Comment by TempleLionStudios** [comment]
I run the toonsquidanimators on reddit 😊 We would love to see your animations there! ⭐️

I also run an "iPad Animators" discord server, but honestly 95% of us use Toonsquid 😉 (invite link below)

It's for animators who do their work on iPads (any tablets are welcome). We have monthly challenges and we share animations, work-in-progress and discuss being an animator ♥️

I only use ToonSquid for my work and would gladly assist anyone who wants to learn it.

I truly have a great love for this app! It allows me to build my life dream of creating an animated universe! Thank you again Keiwan, I can't say this enough 🙏🏽🙏🏽🙏🏽 You litterally changed my life making ToonSquid ⭐️!

https://discord.gg/qeF7QNAd2H

**Reply by Gottafindaname** [reply]
I really recommend this server 👍

**Reply by ZafuMusic** [reply]
Im joining now! :)

**Comment by kavlor** [comment]
I think this should be here as a it is a community based channel.

Helena Risc
https://www.youtube.com/watch?v=QDt-6c8QHi0

**Reply by trinhpxl** [reply]
oh hey i beta tested helenas sketch tube !! it was p fun and awesome :]

**Reply by kavlor** [reply]
Officially up now

https://www.sketchtube.com/
